x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,574 @@
+[[update-node-certs]]
+== Updating node security certificates
+If your cluster certificates are expiring soon or you no longer have the key for
+your existing Certificate Authority (CA), then you'll need to update the TLS
+certificates for both the transport and HTTP layers. When updating your node
+certificates, there are two main scenarios that you'll encounter:
+
+* <<update-node-certs-same,Your CA and key remains the same>>
+* <<update-node-certs-different,Your CA certificate and key changes>>
+
+Regardless of the scenario, {es} monitors the SSL resources for updates
+on a five-second interval, which is the default. You can just copy the new
+certificate and key files (or keystore) into the {es} configuration directory
+and your nodes will detect the changes. 
+
+Because {es} doesn't reload the `elasticsearch.yml` configuration,
+you must use *the same file names* so that {es} refreshes the security
+certificates and keys.
+
+If you need to update the `elasticsearch.yml` configuration or change
+passwords for keys or keystores that are stored in the
+<<secure-settings,secure settings>>, then you must complete a
+<<use-rolling-restarts,rolling restart>>. {es} will not reload changes for
+passwords stored in the secure settings. 
+
+[[use-rolling-restarts]]
+.Rolling restarts are preferred
+****
+While it's possible to do an in-place update for security certificates, using
+a <<restart-cluster-rolling,rolling restart>> on your cluster is safer. An in-place update avoids some
+complications of a rolling restart, but incurs the following risks:
+
+* If you use PEM files, your certificate and key are in separate files. You
+must update both files _simultaneously_ or the node might experience a temporary
+period where it cannot establish new connections.
+* Updating the certificate and key does not automatically force existing
+connections to refresh. This means that even if you make a mistake, a node can
+seem like it's functioning but only because it still has existing connections.
+It's possible that a node will be unable to connect with other nodes, rendering
+it unable to recover from a network outage or node restart.
+****
+
+[[update-node-certs-same]]
+=== Update security certificates with the same CA
+++++
+<titleabbrev>With the same CA</titleabbrev>
+++++
+
+If your CA hasn't changed, you can replace your existing certificate on each
+node with a new certificate that has the same filename. When {es} restarts, it
+reloads the file and uses the new certificate. Using the same filename means
+that you don't have to update the `elasticsearch.yml` file, and can just
+replace the existing certificate.
+
+You don't have to restart each node, but doing so forces new TLS connections and is <<use-rolling-restarts,a recommended practice>> when updating certificates.
+Therefore, the following steps include a node restart after updating each
+certificate.
+
+You'll generate new node certificates and keys for both the transport layer and
```

Comment:
Same comment about _both_ not being necessarily true.

Reply:
We can amend to read:
>The following steps provide instructions for generating new node certificates and keys for both the transport layer and the HTTP layer. You might only need to replace one of these certificates depending on which of your certificates are expiring.

Reply:
one of these layer's certificates most probably but ++ to the essence of the suggested change